_episodes/07-github.md

```diff
@@ -112,22 +119,188 @@ $ git remote -v
+{: .callout}
+
+The first thing we are going to do is check if this has already been done on the computer you’re on.  Because generally speaking, this setup only needs to happen once and then you can forget about it. 
+
+> ## Keeping your keys secure
+> You shouldn't really forget about your SSH keys, since they keep your account secure. It’s good 
+>  practice to audit your secure shell keys every so often. Especially if you are using multiple 
+>  computers to access your account.
+{: .callout}
+
+We will run the list command to check what key pairs already exist on your computer.
+
+~~~
+ls -al ~/.ssh
+~~~
+{: .language-bash}
+
+Your output is going to look a little different depending on whether or not SSH has ever been set up on the computer you are using. 
+
+Dracula has not set up SSH on his computer, so his output is 
+
+~~~
+ls: cannot access '/c/Users/Vlad Dracula/.ssh': No such file or directory
+~~~
+{: .output}
+
+If SSH has been set up on the computer you're using, the public and private key pairs will be listed. The file names are either `id_ed25519`/`id_ed25519.pub` or `id_rsa`/`id_rsa.pub` depending on how the key pairs were set up.  
+
+Since they don’t exist on Dracula’s computer, he uses this command to create them: 
+
+~~~
+$ ssh-keygen -t ed25519 -C "vlad@tran.sylvan.ia"
+~~~
+{: .language-bash}
+
+If you are using a legacy system that doesn't support the Ed25519 algorithm, use:
+`$ ssh-keygen -t rsa -b 4096 -C "your_email@example.com"`
+
+~~~
+Generating public/private ed25519 key pair.
+Enter file in which to save the key (/c/Users/Vlad Dracula/.ssh/id_ed25519):
+~~~
+{: .output}
+
+We want to use the default file, so just press enter.
+
+~~~
+Created directory '/c/Users/Vlad Dracula/.ssh'.
+Enter passphrase (empty for no passphrase):
+~~~
+{: .output}
+
+Now, it is prompting Dracula for a passphrase.  Since he is using his lab’s laptop that other people sometimes have access to, he wants to create a passphrase.  Be sure to use something memorable or save your passphrase somewhere, as there is no "reset my password" option. 
+
+~~~
+Enter same passphrase again:
+~~~
+{: .output}
+
+After entering the same passphrase a second time, we receive the confirmation
+
+~~~
+Your identification has been saved in /c/Users/Vlad Dracula/.ssh/id_ed25519
+Your public key has been saved in /c/Users/Vlad Dracula/.ssh/id_ed25519.pub
+The key fingerprint is:
+SHA256:SMSPIStNyA00KPxuYu94KpZgRAYjgt9g4BA4kFy3g1o vlad@tran.sylvan.ia
+The key's randomart image is:
++--[ED25519 256]--+
+|^B== o.          |
+|%*=.*.+          |
+|+=.E =.+         |
+| .=.+.o..        |
+|....  . S        |
+|.+ o             |
+|+ =              |
+|.o.o             |
+|oo+.             |
++----[SHA256]-----+
+~~~
+{: .output}
+
+The "identification" is actually the private key. You should never share it.  The public key is appropriately named.  The "key fingerprint" 
+is a shorter version of a public key.
```

Comment:
```suggestion
is a [shorter version](https://en.wikipedia.org/wiki/Fingerprint_(computing)) of a public key.
```

Reply:
I think https://en.wikipedia.org/wiki/Public_key_fingerprint would be a more applicable article link for this level.